From a3ceabcdcd82397a6f6be19d7e3f55c534102025 Mon Sep 17 00:00:00 2001
From: Sergey Larin <cerg2010cerg2010@mail.ru>
Date: Thu, 20 Jun 2019 20:08:45 +0300
Subject: [PATCH] ARM: dts: tegra20-glide: Disable BBAT regualtor

This regulator is unused on this board, so disable it.

Signed-off-by: Sergey Larin <cerg2010cerg2010@mail.ru>
---
 arch/arm/boot/dts/tegra20-glide.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/tegra20-glide.dts b/arch/arm/boot/dts/tegra20-glide.dts
index f2af07aba8a3..2e40b051546f 100644
--- a/arch/arm/boot/dts/tegra20-glide.dts
+++ b/arch/arm/boot/dts/tegra20-glide.dts
@@ -819,7 +819,6 @@
 
 				bbat {
 					regulator-name = "bbat";
-					regulator-always-on;
 				};
 
 				sdby {
-- 
2.22.0

